src/main/java/com/jme3/math/Quaternion.java

```diff
@@ -803,110 +803,114 @@ public float toAngleAxis(Vector3f axisStore) {
 		return angle;
 	}
 
+	public float angleBetween(Quaternion q1, Quaternion q2) {
```

Comment:
```suggestion
	public static float angleBetween(Quaternion q1, Quaternion q2) {
```

Reply:
Actually, it should be `public float angleBetween(Quaternion q2)` and q1 should be `this`.